termine/announce.go

```diff
@@ -36,7 +36,7 @@ var (
 func init() {
 	cmdAnnounce.Run = RunAnnounce
 	cmdAnnounce.Flag.StringVar(&targetmailaddr, "address", "treff@noname-ev.de", "Mailadresse, an die Ankündigungen gehen sollen.")
-	cmdAnnounce.Flag.BoolVar(&confirmAnnounce, "confirm", false, "Frage nach Bestätigung bevor die Mail gesendet wird")
+	cmdAnnounce.Flag.BoolVar(&confirmAnnounce, "confirm", true, "Frage nach Bestätigung bevor die Mail gesendet wird")
```

Comment:
How would I use the `termine announce` as cronjob with this change? I think you have to change the usage (and therefore the name) of this flag.